examples/scaling_example/scaling.c

```diff
@@ -289,6 +291,16 @@ run_advanced_rings(struct onvm_nf_info *nf_info) {
         }
 
         while (keep_running && rx_ring && tx_ring && nf) {
+                /* Check for a stop message from the manager. */
+                msg_q = nf->msg_q;
+                if (likely(rte_ring_count(msg_q) > 0)) {
+                        msg = NULL;
+                        rte_ring_dequeue(msg_q, (void **) (&msg));
+                        if (msg->msg_type == MSG_STOP) {
+                                keep_running = 0;
+                        }
```

Comment:
also don't memleak :)
rte_mempool_put(nf_msg_pool, (void *)msg);

Reply:
nf_msg_pool isn't defined here, so I believe I'll have to grab it from onvm_nflib..